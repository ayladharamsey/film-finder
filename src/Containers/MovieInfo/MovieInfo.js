import React from 'react';
//map state to props for the ones that you want in your store

export const MovieInfo = () => {
    // need to access store and render data
    // return (

    // )
    return (
        <>
        </>
    )
}

export default MovieInfo;