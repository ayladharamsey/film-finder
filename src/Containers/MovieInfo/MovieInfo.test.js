import React from 'react';
import { shallow } from 'enzyme';
import { MovieInfo } from './MovieInfo';

describe('MovieInfo container tests', () => {

  it('Movie should match its snapshot', () => {
    const wrapper = shallow( <MovieInfo/> )
    expect(wrapper).toMatchSnapshot();
  })
})