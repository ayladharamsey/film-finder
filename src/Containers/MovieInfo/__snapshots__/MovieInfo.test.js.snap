// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieInfo container tests Movie should match its snapshot 1`] = `ShallowWrapper {}`;
